src/components/FormTodo/FormTodo.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { Todo, User } from '../../types';
+
+interface Props {
+  onAdd: (todo: Todo) => void;
+  user: User[];
```

Comment:
The `user` prop is defined as an array of `User` objects. Consider renaming this prop to `users` to better reflect its nature as an array and avoid confusion.